map/mwm_url.cpp

```diff
@@ -251,8 +241,7 @@ void ParsedMapApi::ParseMapParam(std::string const & key, std::string const & va
   }
   else if (key == kZoomLevel)
   {
-    double zoomLevel;
-    if (strings::to_double(value, zoomLevel))
+    if (double zoomLevel; strings::to_double(value, zoomLevel))
```

Comment:
1. Because it does not leak the variable outside of its usage scope.
2. It takes fewer lines of code and allows one to see more useful code on the screen, thus making code review easier for a reviewer.
3. It teaches contributors to use safe/better C++17 patterns that they may not be aware of.

Reply:
As mentioned above I also don't like this syntax. This is not a place when scope matters to put several expressions into if.
Probably, here is the only way it looks natural: ```if (auto p = foo(); p)```
I can't prohibit writing like this but don't want to get my code modified like this.